holoviews/operation/resample.py

```diff
@@ -114,6 +114,14 @@ def instance(self_or_cls,**params):
         inst._precomputed = {}
         return inst
 
+
+    def __call__(self, element, **kwargs):
+        opts = element.opts.get('plot', 'bokeh')
+        opts = {} if opts is None else opts.kwargs
+        if 'multi_y' in opts:
```

Comment:
```suggestion
        if 'multi_y' in opts and isinstance(element, CompositeOverlay):
```

Reply:
Not sure that's quite right, but I don't see a good reason to warn if you just provide an Element. Also the `multi_y` may be set after the fact and then this warning won't do anything. So overall I don't think this approach is very good.

Reply:
Fair enough. Where would you warn from when using datashader with `multi_y=True` then?

Reply:
One other alternative of course is to not warn at all and to document this instead...